frontend/src/app/modules/boards/board/board-list/board-list.component.ts

```diff
@@ -403,12 +404,15 @@ export class BoardListComponent extends AbstractWidgetComponent implements OnIni
 
     observable
```

Comment:
A parent component sets state dependent on this state. In my opinion, that means that this action should be done in the parent that needs the state. So instead of emitting up `setContainerClass`, the parent should be responsible for this action and all state that relates to it, and then handles classes etc. internally. Any state that needs to reach this component should be handed over via inputs.

Reply:
@aleixsuau I don't know if you see this differently, but the fact that we need to `emit` a class change to the parent shows me the state is in the wrong place.

Reply:
Alternatively, if the action really is _jsut about this component_, then the class should also be set inside of this component, and the CSS should be refactored.

Reply:
Hi @b12f this style is not related to the board-list component itself. In the board-list component, we only create a list but In the board-list-container component, we are creating several board-list components with additional functionalities, if one of these lists has an error, it shouldn't be shown. The parent element (container of each list) now should be draggable and has some other functionalities, so we should hide the whole container element of the failed list. And the state should be checked for each list, not for the whole parent component. If we want to handle this CSS style in the board-list component, we should add dragging and other functionalities into this component, that is not actually part of it.  In another way, in my opinion, handling rendering the data of children is not related to the parent component and it should be checked in the child component.